repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/acceptance-tests/series-with-publication-with-datasets.yaml

```diff
@@ -1,5 +1,96 @@
 ---
 /content/documents/corporate-website/publication-system/acceptance-tests/series-with-publication-with-datasets:
+  /content1:
```

Comment:
Have you intended to add this? Don't think name of content1 is correct and this is a series not an archive